proposal/xskipper.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Xskipper data skipping for spark
+---
+
+Xskipper is An Extensible [Data Skipping](https://xskipper.io/master/concepts/data-skipping) Framework, it provides a library for creating, managing and deploying data skipping indexes with Apache Spark.
+
+- Boosts performance and reduce cost by skipping over irrelevant data
+- Supports Parquet, CSV, JSON, ORC and Avro
+- Hive tables supported
+- Supports MinMax, ValueList and BloomFilter indexes out of the box
+- Supports data skipping for User Defined Functions
+- Easy to add additional index types
+
+Xskipper is a Spark extension, [open source project](https://github.com/xskipper-io/xskipper), with scala and python api. To get started boosting sql queries with spark, get xskipper jar that fits your spark version and add it to spark packages jars.
+
+## Architecture
+[Xskipper Data Skipping Architecture](https://xskipper.io/master/concepts/extensible/)
+
+## User Use Cases
+Spark big data users who want to boost performance and reduce cost by indexing their data and skipping over irrelevant data
+
+## Implementation
+Xskipper is a Spark extension with [scala and python api](https://xskipper.io/master/api/indexing/). 
+Xskipper jar for the relevant spark version should be added to pyspark packages jars, once the jar is included data can be indexed and dataskipping can be enabled. See [example notebook] (https://github.com/xskipper-io/xskipper/blob/master/notebooks/python/Xskipper%20-%20Python%20Sample.ipynb).
```

Comment:
Please remove the whitespace after `[example notebook]`, it is breaking the link in markdown